crates/but-gitea/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::{Context as _, Result};

```

Comment:
The module annotations should be:
```suggestion
#![forbid(unsafe_code)]
#![deny(missing_docs)]
use anyhow::{Context as _, Result};
```